### PR TITLE
fix wrong EmailHeader type definition

### PR DESCRIPTION
**Pull request by qyb**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -253,6 +256,7 @@ export interface IEmailProperties {
   createdModSeq: number;
   updatedModSeq: number;
   receivedAt: IUtcDate;
+  headers?: EmailHeader[];
```
Comment (chibenwa) [inline src/types.ts:259]: Maybe

```suggestion
  headers: EmailHeader[] | null;
```

To be consistent?

**Review by chibenwa — DISMISSED** [review]
Thanks for the contribution ;-)

For me that is good as it is but adding a test would be even better.

Regards

File: src/types.ts (hunk 1 above)
Comment (qyb) [reply src/types.ts:259]: Yes, you are right.

**Comment by qyb** [comment]
> Thanks for the contribution ;-)
> 
> For me that is good as it is but adding a test would be even better.
> 
> Regards

Which JMAP server are you using? Recently I began to implement webmail for cyrus-imapd (demo: https://bwh1.rubyfish.app/ ). I found some limitations when I try jmap-client-ts. I want to put my patch into the mainstream in the following days.

**Comment by chibenwa** [comment]
> Which JMAP server are you using? 

We are using Apache James https://james.apache.org/

> I found some limitations when I try jmap-client-ts. I want to put my patch into the mainstream in the following days.

I would be interested to know wich ones.

Of course this library should have support for Cyrus too and your patches are more than welcome.

**Comment by qyb** [comment]
> > Which JMAP server are you using?
> 
> We are using Apache James https://james.apache.org/
> 
> > I found some limitations when I try jmap-client-ts. I want to put my patch into the mainstream in the following days.
> 
> I would be interested to know wich ones.
> 
> Of course this library should have support for Cyrus too and your patches are more than welcome.

I noticed CONTRIBUTING.md just a minute ago. I'll try the docker image & test suite.

**Comment by alagane** [comment]
Hi, thanks for your contribution. 
Looks good to me, I checked the specs and it is ok, you just have to apply the modification @chibenwa suggested (either by committing it or by force-pushing it).

**Review by alagane — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Pull request merged**